packages/lesswrong/components/ea-forum/EAHome.tsx

```diff
@@ -13,12 +13,42 @@ const showHandbookBannerSetting = new DatabasePublicSetting<boolean>('showHandbo
 const showEventBannerSetting = new DatabasePublicSetting<boolean>('showEventBanner', false)
 const showMaintenanceBannerSetting = new DatabasePublicSetting<boolean>('showMaintenanceBanner', false)
 
+/**
+ * Build structured data to help with SEO.
+ */
+const getStructuredData = () => ({
+  "@context": "http://schema.org",
+  "@type": "WebSite",
+  "url": `${getSiteUrl()}`,
+  "potentialAction": {
+    "@type": "SearchAction",
+    "target": `${getSiteUrl()}search?query={search_term_string}`,
```

Comment:
It's probably better to use `combineUrls` for the concatenation here (which is in the same file as `getSiteurl`) just to make sure nothing weird happens with slashes.